cmd/ocm/create/cluster/cmd.go

```diff
@@ -1334,6 +1339,10 @@ func promptExistingVPC(fs *pflag.FlagSet, connection *sdk.Connection) error {
 	return err
 }
 
+func promptClusterPrivacy(fs *pflag.FlagSet, connection *sdk.Connection) error {
+	return arguments.PromptBool(fs, "private")
```

Comment:
Considering that this string value must match the string value of the flag definition, it would be an improvement to maintainability if it were a const instead of a hard-coded value.